### PR TITLE
Update Turkish translation

### PR DESCRIPTION
**Pull request by libreajans**

### CODE REVIEW

File: locales/tr.po [hunk 1]
```diff
@@ -1,95 +1,105 @@
...
-msgstr ""
+msgstr "Girdi dosyasının yanında (→ resim.opti.png)"
 
+# Optimized kısmını çevirmeyin!
 #. Do not translate the 'optimized/' part of the example file path, it is not localized
 #: yoga_image_optimizer/data/ui/main-window.glade:222
 #: yoga_image_optimizer/data/ui/settings-window.glade:224
 msgid "In a subfolder (→ optimized/image.png)"
-msgstr ""
+msgstr "Alt klasörde (→ optimized/image.png)"
```
Comment (flozz) [inline locales/tr.po:68]: Maybe this should be translated like this to be consistent with `resim.opti.png` above?
```suggestion
msgstr "Alt klasörde (→ optimized/resim.png)"
```

**Comment by flozz** [comment]
Hello,

Thank you for your contribution. It is great to have a complete Turkish translation!

I only have an interrogation about `optimized/image.png` that may be translated to `optimized/resim.png` to be consistent with `resim.opti.png`. :)

File: locales/tr.po (hunk 1 above)
Comment (libreajans) [reply locales/tr.po]: You are correct. I changet it to -> resim.opti.png

**Comment by flozz** [comment]
Thank you! :)

**Pull request merged**